app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Base
+  has_many :tweets
+  has_secure_password
+
```

Comment:
It might also be useful to add more validations to the User model, such as presence and uniqueness for username and email:
```
validates :username, :email, presence: true
validates :username, :email, uniqueness: true
```
Then, in controller actions that create or update these attributes, we can use `@user.valid?` or `if @user.save` to control the next steps.